lib/optionsMenu.js

```diff
@@ -0,0 +1,138 @@
+const { LitElement, html, css } = require('lit-element')
+require('./github_link')
+require('./components/button')
+require('./components/buttonlink')
+require('./components/textfield')
+
+class OptionsMenu extends LitElement {
+  constructor () {
+    super()
+    this.inMenu = false
+  }
+
+  disableOptionsMenu () {
+    this.inMenu = false
+    this.style.display = 'none'
+  }
+
+  enableOptionsMenu () {
+    this.inMenu = true
+    document.exitPointerLock()
+    this.style.display = 'block'
+    this.focus()
+  }
+
+  static get styles () {
+    return css`
+    :host {
+      --guiScale: var(--guiScaleFactor, 3);
```

Comment:
I can't help but not like this, there are such better ways to do css other than this

Reply:
This has been what we’ve been doing for all the other components, from the login screen to the hotbar to the existing game menu. Don’t see any reason to change it in this PR specifically.